clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -165,7 +167,9 @@ public static void displayHelp(Options options) {
 
   /** Executes the appropriate command based on the command type. */
   private void executeCommand() {
-    if (entity.equals(CommandEntities.COLUMN)) {
+    if (line.hasOption(GravitinoOptions.OWNER)) {
```

Comment:
It seems the "--owner" option has the first priority, once it appears, the logic will go to owner related commands. But why it got so high priority?

Reply:
The owner command, unlike other commands, is done via a flag, not as an entity. This is needed as we need to know the entity type to update, and because of this, it needs to be checked first.

Reply:
Alternatively, if you do not want the check here, we could duplicate the code in every single handle command method, but that seems error prone to me.